src/core/sig_state.ml

```diff
@@ -200,14 +200,14 @@ let open_sign : sig_state -> Sign.t -> sig_state = fun ss sign ->
   in
   {ss with in_scope; builtins; unops; binops; pp_hints}
 
-(** Dummy [sig_state] made from the dummy signature. *)
+(** [dummy : sig_state] made from the dummy signature. *)
```

Comment:
no need to put the type in the comment

Reply:
In general, I don't think we need to follow the pattern `[fun args]` with constants, we just have to explain what they are, hence the first version of the comment.

Reply:
Done